### DISCUSSION TITLE
Szállítólevél típusú bizonylatok

### DISCUSSION THREAD

**Discussion by bvidor** [nav-gov-hu/eRECEIPT#12] [Ötletek]
A vendéglátásban gyakori, a társ rendszerek közötti átterhelés, például az éttermi szobahitel. Ezen bizonylatoknak forgalomként, meg kell jelenjenek a vendéglátásban, de kiegyenlíteni majd a szállodai rendszerben fogják.

**Comment by renced42** [comment]
Kedves @bvidor 
Mi a kérdés?

**Comment by bvidor** [comment]
Tehát a kérdés az, hogy számíthatunk-e valamilyen speciális bizonylat típusra (szállítólevél), vagy ezek a bizonylatok nem képzik az eNyugta rendszer felé küldendő adatokat?
A szobahiteles bizonylatokat például a vendégek aláírják, hogy a későbbi szállodai számlában szerepeltetve ne legyen kérdéses a fogyasztás, ezeket jelenleg az AEE-k vagy fogyasztási összesítőnek vagy szabad nyomtatásként kapják meg.

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
